doc/python/bar-charts.md

```diff
@@ -304,6 +304,158 @@ fig.update_layout(
 fig.show()
 ```
 
+### Control Bar Position in Different Subplots
+
+To control bars positional range among several subplots, set the same axes to the same [alignmentgroup](https://plot.ly/python/reference/#bar-alignmentgroup). In the following example we have two subplots sharing an x axis with two bar traces (trace0, trace1) on the top, and one bar trace (trace2) on the bottom, that all are aligned by setting the same `alignmentgroup`. 
+You also can line up bars of the same positional coordinate by setting [offsetgroup](https://plot.ly/python/reference/#bar-offsetgroup).
```

Comment:
Bar position can also be coordinated by placing traces in the same [offsetgroup](https://plot.ly/python/reference/#bar-offsetgroup).